src/resmom/mom_inter.c

```diff
@@ -105,8 +105,11 @@ int    amt;
 			total += got;
 		} else if (got == 0)
 			break;
-		else
+		else {
+			log_eventf(PBSEVENT_DEBUG3, PBS_EVENTCLASS_SERVER, LOG_DEBUG, __func__,
```

Comment:
this is an error, why are you using debug3?

Reply:
> this is an error, why are you using debug3?

I agree. I have changed it to PBSEVENT_ERROR at all places.